tests/lib/tools/lxd-state

```diff
@@ -32,6 +33,33 @@ prepare_snap(){
     fi
 }
 
+setup_proxy() {
```

Comment:
Could we instead modify the default profile so that this is automatic for all instances?

Reply:
included in launch function